tests/util_tests/test_state_helpers.py

```diff
@@ -75,6 +106,8 @@ def test_state_root_override_enter(mockReadPVFile, mockReadInstrumentConfig, moc
     mockDefaultGroupingMapPath.return_value = Path(Resource.getPath("inputs/pixel_grouping/defaultGroupingMap.json"))
 
     stateId = "ab8704b0bc2a2342"
+    decodedKey = "some_decoded_key"
+    mockGenerateStateId.return_value = (stateId, decodedKey)
```

Comment:
I strongly recommend that you use `None` here as the decoded key (as the "decoded" part is optional).  Otherwise the actual key will be invalid, which might screw someone up in our testing future.

Reply:
Fixed!